services/squid-egress-proxy/build.yaml

```diff
@@ -0,0 +1,30 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
```

Comment:
Please add a namespace to these objects

Reply:
since we want this image to be available to any ci namespace, is it alright that it is set to `openshift` like most other IS?